run_dir/static/js/deliveries.js

```diff
@@ -325,29 +256,34 @@ $('.button-save-bioinfo-responsible').click(function() {
     $(this).parent().find('.button-reset-bioinfo-responsible').hide();
 });
 
-var categories={ 'Workset': 'label-primary',
-                            'Flowcell': 'label-success',
-                            'Meeting': 'label-info',
-                            'Decision': 'label-info',
-                            'User Communication': 'label-danger',
-                            'Bioinformatics': 'label-warning',
-                            'All': 'label-default' }
+var categories={ 'Workset': 'bg-primary',
+                 'Flowcell': 'bg-success',
+                 'Decision': 'bg-info',
+                 'Lab': 'bg-success',
+                 'Bioinformatics': 'bg-warning',
+                 'User Communication': 'bg-usr',
+                 'Administration': 'bg-danger',
+                 'Important': 'bg-imp',
+                 'Deviation': 'bg-devi' }
 
 // not using running_notes.js anymore
 // Insert new running note and reload the running notes table
 $("#notes_form").submit( function(e) {
     e.preventDefault();
     // should be set when clicking the button 'See All'
     var project_id = $('#runningNotesModalDeliveries_title').attr('data-project-id');
-
     var text = $('#new_note_text').val().trim();
-    text = $('<div>').text(text).html();
-    var category = $('#rn_category option:selected').val();
+    var category = $('.rn-categ button.active').text().trim();
     if (text.length == 0) {
         alert("Error: No running note entered.");
         return false;
     }
-    // don't like to use this
+    if (!$('.rn-categ button.active').text()) {
+       if (!confirm("Are you sure that you want to submit without choosing a category?")) {
+          return false;
+       }
+    }
+    //don't like to use this
```

Comment:
What's this about? The confirm part above or the ajax below?

Reply:
No idea, if you're asking you about the comment. It was already in the code. Do you want me to remove it?

Reply:
If we don't know what its for and we don't agree, then yes remove it

Reply:
Done!